src/main/uploader.ts

```diff
@@ -95,6 +82,35 @@ export class MinidumpUploader {
     }/api/${projectId}/minidump?sentry_key=${user}`;
   }
 
+  /**
+   * Create minidump request to dispatch to the transpoirt
+   */
+  private async _toMinidumpRequest(event: Event, minidumpPath: string): Promise<SentryElectronRequest> {
+    const envelopeHeaders = JSON.stringify({
+      event_id: event.event_id,
+      sent_at: new Date(timestampWithMs() * 1000).toISOString(),
```

Comment:
It's probably worth noting that every time you retry a request, you have to generate a new `sent_at` timestamp. Otherwise, clock drift correction will see an old timestamp and assume your clock is lagging behind.

It seems that you do this correctly, can you confirm? I'd suggest to add a comment at some appropriate place for the future.